src/main/java/edu/harvard/iq/dataverse/DatasetFieldValue.java

```diff
@@ -85,6 +88,31 @@ public String getValueForEdit() {
     public void setValueForEdit(String value) {
         this.value = value;
     }
+    
+    public String getDisplayValue() {
+        String retVal = "";
+        if (!StringUtils.isBlank(this.getValue()) && !DatasetField.NA_VALUE.equals(this.getValue())) {
+            String format = this.datasetField.getDatasetFieldType().getDisplayFormat();
+            if (StringUtils.isBlank(format)) {
+                format = "#VALUE";
+            }           
+            String sanitizedValue = !this.datasetField.getDatasetFieldType().isSanitizeHtml() ? this.getValue() :  MarkupChecker.sanitizeBasicHTML(this.getValue());    
+            
+                if (!this.datasetField.getDatasetFieldType().isSanitizeHtml() && this.datasetField.getDatasetFieldType().isEscapeOutputText()){
```

Comment:
See other comment about possible new field type to reduce render logic complexity.  May be more work upfront but less work/debugging in the future